mobile-app/app/screens/AppNavigator/screens/Portfolio/screens/ConvertScreenV2.tsx

```diff
@@ -0,0 +1,401 @@
+      light={tailwind('border-mono-light-v2-300')} dark={tailwind('border-mono-dark-v2-300')}
+    >
+      <ThemedViewV2
+        style={tailwind('flex-row items-center pb-5')}
+      >
+        <ThemedTextV2
+          style={tailwind('font-normal-v2 text-sm pr-2')} testID='convert_available_label'
+          light={tailwind('text-mono-light-v2-500')} dark={tailwind('text-mono-dark-v2-500')}
+        >
+          {`${translate('screens/ConvertScreen', 'Available {{unit}}', { unit: props.unit })}`}
+        </ThemedTextV2>
+        <NumberFormat
+          decimalScale={8}
+          displayType='text'
+          renderText={(value) => (
+            <ThemedTextV2
+              style={tailwind('flex-1 font-normal-v2 text-sm text-right')}
+              light={tailwind('text-mono-light-v2-800')} dark={tailwind('text-mono-dark-v2-800')}
+              testID='convert_available_amount'
+            >
+              {value}
+            </ThemedTextV2>
+          )}
+          thousandSeparator
+          value={new BigNumber(props.oriTargetAmount).toFixed(8)}
+        />
+      </ThemedViewV2>
+      <ThemedViewV2
+        style={tailwind('flex-row items-center pt-5 border-t-0.5')}
+        light={tailwind('border-mono-light-v2-300')} dark={tailwind('border-mono-dark-v2-300')}
+      >
+        <ThemedTextV2
+          style={tailwind('font-normal-v2 text-sm pr-2')} testID='convert_resulting_label'
+          light={tailwind('text-mono-light-v2-500')} dark={tailwind('text-mono-dark-v2-500')}
+        >
+          {`${translate('screens/ConvertScreen', 'Resulting {{unit}}', { unit: props.unit })}`}
+        </ThemedTextV2>
+        <NumberFormat
+          decimalScale={8}
+          displayType='text'
+          renderText={(value) => (
+            <ThemedTextV2
+              style={tailwind('flex-1 font-semibold-v2 text-sm text-right')}
+              light={tailwind('text-mono-light-v2-800')} dark={tailwind('text-mono-dark-v2-800')}
+              testID='convert_result_amount'
+            >
+              {value}
+            </ThemedTextV2>
+          )}
+          thousandSeparator
+          value={props.totalTargetAmount}
+        />
+      </ThemedViewV2>
+    </ThemedViewV2>
+  )
+}
+
+function canConvert (amount: string, balance: string): boolean {
+  return new BigNumber(balance).gte(amount) && !(new BigNumber(amount).isZero()) && (new BigNumber(amount).isPositive())
+}
+
+function getConvertibleUtxoAmount (mode: ConversionMode, source: AddressToken): string {
+  if (mode === 'accountToUtxos') {
+    return source.amount
+  }
+
+  const utxoToReserve = '0.1'
+  const leftover = new BigNumber(source.amount).minus(new BigNumber(utxoToReserve))
+  return leftover.isLessThan(0) ? '0' : leftover.toFixed()
+}
+
+function isUtxoToAccount (mode: ConversionMode): boolean {
+  return mode === 'utxosToAccount'
+}
+
+export function getDisplayUnit (unit: 'UTXO' | 'Token'): 'UTXO' | 'tokens' {
```

Comment:
We can also have an enum or type for 'UTXO' | 'Token', I saw it used somewhere also

Reply:
done, updated using enum